saleor/graphql/shipping/types.py

```diff
@@ -305,60 +306,51 @@ class Meta:
         )
 
     @staticmethod
-    def resolve_type(root: ChannelContext[models.ShippingMethod], info, **_kwargs):
-        return root.node.type
+    def resolve_type(root: ShippingMethodData, info, **_kwargs):
```

Comment:
Do you still need these resolvers? Maybe I am missing something, but default graphene logic should be able to handle simple resolvers if the field name matches the field in the object.

Reply:
You are right, previously unit tests were failing, but this could have been due to some legacy ChannelContext logic. I've removed all those obsolete resolvers